src/Microsoft.Framework.PackageManager/Install/InstallGlobalCommand.cs

```diff
@@ -47,7 +47,7 @@ public FeedOptions FeedOptions
 
             if (packageIdAndVersion == null)
             {
-                WriteError("The name of the package to be installed was not specified.");
+                WriteError($"Installing {packageId} failed");
```

Comment:
Good catch, should we just say "Installing failed" instead? They should know what they are trying to install